include/log.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <ostream>
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <unordered_map>
+#include <variant>
+
+// TODO: Make this configurable through CMake?
+#define SPDLOG_ACTIVE_LEVEL SPDLOG_LEVEL_TRACE
+#include <spdlog/spdlog.h>
+
+#include <spdlog/fmt/ostr.h>
+#include <spdlog/sinks/stdout_color_sinks.h>
```

Comment:
I don't think any of

- `<ostream>`
- `<sstream>`
- `<spdlog/fmt/ostr.h>`
- `<spdlog/sinks/stdout_color_sinks.h>`

are actually used in this file.

Reply:
Thanks, I've removed (hopefully) all superfluous includes here and in the other files.